Source/Charts/Jobs/ViewPortJob.swift

```diff
@@ -17,11 +17,11 @@ import CoreGraphics
 open class ViewPortJob: NSObject
 {
     @objc internal var point: CGPoint = CGPoint()
-    @objc internal weak var viewPortHandler: ViewPortHandler?
+    @objc internal weak var viewPortHandler: ViewPortHandler? // TODO: Can this be unowned?
```

Comment:
before merge, do we need to finish TODO? I am willing to check if we can use `unowned` when I got time, but seems weak is totally fine here. What's your concern using weak?

Reply:
`weak` does work here, but I think what is actually intended is `unowned`. In terms of function, nothing would change, but it does clean up a lot of `guard` statements if we can use `unowned` I feel like that is a separate PR and one that requires a bit more digging. I'm not sure what the policy is on TODOs

Reply:
I don't know the policy neither. but if it is another PR, could you remove the comment?  I will merge later.

Reply:
Done